AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -7,6 +7,7 @@
 !
 ! SECTION: Popups - Regular rules
 !
+kurier.de,np-coburg.de,frankenpost.de##div[class^="cleverpush-"]
```

Comment:
Can't reproduce. Popups filter blocks by `||cleverpush.com^$third-party,domain=~app.bsdex.de`

Reply:
Same.

Reply:
Ah, you are right, thanks, I just tested this site without Annoyances filter
Will remove the redundant rule